pkgs/os-specific/linux/kernel/perf.nix

```
@@ -54,8 +60,10 @@ stdenv.mkDerivation {
     "-Wno-error=stringop-truncation"
   ];
 
-  postPatch = ''
+  postPatch = if (kernel.kernelOlder "5.13") then ''
     patchShebangs scripts/bpf_helpers_doc.py
+  '' else ''
+    patchShebangs scripts
```

Any harm in doing this?

```suggestion
  postPatch = ''
    patchShebangs scripts
  '';
```

---

probably not

---

applied